src/store/userStore.js

```diff
@@ -1,6 +1,11 @@
 import create from 'zustand';
 
 const useUserStore = create((set) => ({
+  userId: localStorage.getItem('userId') || '',
```

Comment:
can we change it to -> `rc_uid` ? Also can you take all the keyed items and make them as a constant and export it from a file called `constant.js` inside lib?

Something like,
```
export const RC_LOCAL_USER_ID = 'rc_uid'
export const RC_LOCAL_USER_NAME = 'rc_uname'
```
and use them everywhere instead of hardcoded strings, this will surely help us in the long run

Reply:
Yeah sure 😊

Reply:
you want me to use 
`export const RC_LOCAL_USER_ID = 'rc_uid';
export const RC_LOCAL_USER_NAME = 'rc_uname';
export const RC_LOCAL_USER_AVATAR_URL = 'rc_avatar_url';
export const RC_LOCAL_USER_TOKEN = 'rc_token';
`
these in every single file of whole project ?

Reply:
I think `LOCAL` must be used when we are storing something locally (in this case, localstorage). But we aren't storing rc_token and rc_uid

so we can have,
```
RC_USER_TOKEN_COOKIE='rc_token'
RC_USER_ID_COOKIE='rc_uid'
```

and for the rc_uid that we are storing keep it,
`RC_LOCAL_USER_ID`

so where we are using localstorage to access userid, use -> LOCAL_USER_ID but where we are using the cookie -> use the cookie string